frontend/utilities/helpers.ts

```diff
@@ -613,7 +615,7 @@ export const humanHostLastRestart = (
 };
 
 export const humanHostLastSeen = (lastSeen: string): string => {
-  if (!lastSeen || lastSeen < "2016-07-28T00:00:00Z") {
+  if (!lastSeen || lastSeen < INITIAL_FLEET_DATE) {
```

Comment:
Should still present never  if before `export const INITIAL_FLEET_DATE = "2016-07-28T00:00:00Z";`